2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,20 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(arr) {
+  const first5Elements = arr.slice(0, 5);
+  return first5Elements;
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(arr) {
+  const newArray = arr.slice();
+  newArray.sort((a, b) => a - b);
+  return newArray;
```

Comment:
Again, not mandatory but you can write `return arr.slice().sort(...)`. You don't want to do it too much generally if your code is going to become too complex to read, but in this case the result is simple enough.